src/transaction.rs

```diff
@@ -353,7 +353,7 @@ impl TxOut {
         if iter.next() == Some(Instruction::Op(opcodes::All::OP_RETURN)) {
             for push in iter {
                 match push {
-                    Instruction::Op(op) if op as u8 > opcodes::All::OP_PUSHNUM_16 as u8 => {}
+                    Instruction::Op(op) if op as u8 > opcodes::All::OP_PUSHNUM_16 as u8 => return false,
```

Comment:
Can we add an `Instruction::Op(op) if op as u8 <= opcodes::All::OP_PUSHNUM_16 as u8 => return true` case. I find it hard to see what isan Op or PushBytes. I think OP_1 aka OP_PUSHNUM_1 is not PushBytes and therefore would result in false here.

Reply:
Opcodes 0 through 76 are "push this many bytes", followed by the 3 PUSHDATA opcodes, a couple reserved ones, then the numbers. Everything above OP_16 is control flow and OP_RETURN. OP_PUSHNUM_1 0x51 is definitely belowe OP_PUSHNUM_16 0x60.

I copied this logic from https://github.com/bitcoin/bitcoin/blob/master/src/script/script.cpp#L243 btw.

Reply:
Oh, I understand what you're saying! Will fix.